models/intel/person-detection-0200/accuracy-check.yml

```diff
@@ -0,0 +1,19 @@
+models:
```

Comment:
@eaidova Is this how config should look like now?

Reply:
@Ilya-Krylov , yes, config structure is correct